contracts.md

```diff
@@ -78,6 +78,11 @@ Likewise, if we want to replace our underlying cache technology (Memcached) with
     class Repository
     {
         /**
+         * The cache.
```

Comment:
I just copied the exact same thing that was on the earlier example on the page to make it consistent.